english/soar/soar.md

```diff
@@ -3,4 +3,8 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Sing?](english/sing/sing.md)
+[Check out Facebook?](https://www.facebook.com)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.